mods/bucket/init.lua

```diff
@@ -116,6 +116,10 @@ function bucket.register_liquid(source, flowing, itemname, inventory_image, name
 				return ItemStack("bucket:bucket_empty")
 			end
 		})
+		if itemname:sub(1,1) == ":" then
+			-- If the itemname starts with a colon, correct that
+			bucket.liquids[source]["itemname"] = itemname:sub(2)
```

Comment:
Why is it necessary to overwrite this value? Couldn't you assign the correct value directly in L57?

e.g. `itemname = [itemname](itemname:sub(1,1) == ":" and itemname:sub(2) or itemname` (or perhaps a less dense version)

Reply:
Ok, I did as recommended.